DelvUI/Interface/NinjaHudWindow.cs

```diff
@@ -339,7 +339,7 @@ public class NinjaHudConfig : PluginConfigObject
 
         [Checkbox("Show Suiton Bar")]
         [CollapseControl(20, 3)]
-        public bool ShowSuitonBar = true;
+        public bool ShowSuitonBar = false;
```

Comment:
I would personally generally avoid unrelated changes, such as change the default for a particular value. Not necessarily because it's a bad thing, but because such changes can be merged much more quickly if they are separate.